checker/check-response-pattern-added-or-changed_test.go

```diff
@@ -24,7 +24,7 @@ func TestResponsePropertyPatternChanged(t *testing.T) {
 	require.Equal(t, checker.ApiChange{
 
 		Id:          "response-property-pattern-changed",
-		Text:        "the response property 'data/created' pattern changed from '^[a-z]+$' to '^(?:([a-z]+-)*([a-z]+)?)$' for the status '200'",
+		Text:        "the 'data/created' response property pattern was changed from '^[a-z]+$' to '^(?:([a-z]+-)*([a-z]+)?)$' for the status '200'",
```

Comment:
Please change:
the %s response property
To:
the %s response property's


Like the other messages, such as:
the %s response property's maxLength was unset from %s for the response status %s